indico/modules/events/timetable/client/js/reducers.ts

```diff
@@ -55,13 +55,36 @@ export default {
         return {...state, changes: [{entries: layoutDays(dayEntries), unscheduled}]};
       }
       case actions.MOVE_ENTRY: {
-        const date = action.date;
-        const newEntries = Object.fromEntries(
-          Object.entries(state.changes[state.currentChangeIdx].entries).map(([day, dayEntries]) => [
-            day,
-            day === date ? action.entries : dayEntries,
-          ])
-        );
+        const {date, entry, entries} = action;
+
+        let entryId: number, newDayEntries: TopLevelEntry[];
+        if (entry.sessionBlockId) {
+          const parentId = entries.findIndex(e => e.id === entry.sessionBlockId);
+          const parent = entries[parentId];
```

Comment:
Why not just do find instead of findIndex? Then you can have one variable called `parent`